lib/features/mailbox/domain/model/subscribe_mailbox_request.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:equatable/equatable.dart';
+import 'package:jmap_dart_client/jmap/mail/mailbox/mailbox.dart';
+import 'package:tmail_ui_user/features/mailbox/domain/model/mailbox_subscribe_action_state.dart';
+import 'package:tmail_ui_user/features/mailbox/domain/model/mailbox_subscribe_state.dart';
+
+class SubscribeMailboxRequest with EquatableMixin {
+
+  final MailboxId mailboxId;
+  final MailboxSubscribeState newState;
+  final MailboxSubscribeStateAction mailboxSubscribeStateAction;
```

Comment:
what is the different?

Reply:
MailboxSubscribeState :
- Disable/enable
MailboxSubscribeStateAction :
- Subscribing/Undo

Reply:
why we need all of these?